isotip.js

```diff
@@ -89,17 +89,18 @@ module.exports = {
       }
 
       var trigger = evt.target || evt.srcElement
+      var target = self.selfOrAncestorByClass(trigger, 'tooltip-click')
```

Comment:
It seems like the `self.hasParent` call on line 109 could just be moved up rather than calling this function. This one can return null, which would prevent the if block on line 103 from running. Alternatively, we could ditch `self.hasParent` in favor of `self.selfOrAncestorByClass`.